spacy/tests/test_requirements.py

```diff
@@ -0,0 +1,59 @@
+import re
+from pathlib import Path
+
+
+def test_build_dependencies(en_vocab):
+    # Check that library requirements are pinned exactly the same across different setup files.
+    libs_ignore_requirements = ["pytest", "pytest-timeout", "mock", "flake8", "jsonschema"]
+    libs_ignore_setup = ["fugashi", "natto-py", "pythainlp"]
+
+    # check requirements.txt
+    root_dir = Path(__file__).parent.parent.parent
```

Comment:
I think if the file is included in the package, it will be at the package root, so:

```suggestion
    root_dir = Path(__file__).parent.parent
```

Reply:
Hm. But then it wouldn't run locally ?